<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="admin-PC" name="techfetch.Techfetch2" tests="8" failures="7" timestamp="3 Aug 2017 08:55:50 GMT" time="72.760" errors="0">
  <testcase name="update" time="40.756" classname="techfetch.Techfetch2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:446)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementsByCssSelector(RemoteWebDriver.java:505)
	at org.openqa.selenium.By$ByCssSelector.findElements(By.java:441)
	at org.openqa.selenium.remote.RemoteWebDriver.findElements(RemoteWebDriver.java:398)
	at org.openqa.selenium.remote.RemoteWebDriver$RemoteTargetLocator.frame(RemoteWebDriver.java:966)
	at techfetch.Techfetch2.update(Techfetch2.java:452)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- update -->
  <testcase name="login" time="32.004" classname="techfetch.Techfetch2"/>
  <testcase name="listing" time="0.000" classname="techfetch.Techfetch2"/>
  <testcase name="upload" time="0.000" classname="techfetch.Techfetch2"/>
  <testcase name="forward" time="0.000" classname="techfetch.Techfetch2"/>
  <testcase name="logout" time="0.000" classname="techfetch.Techfetch2"/>
  <testcase name="fetch" time="0.000" classname="techfetch.Techfetch2"/>
  <testcase name="features" time="0.000" classname="techfetch.Techfetch2"/>
</testsuite> <!-- techfetch.Techfetch2 -->
